packages/google_maps_flutter/google_maps_flutter_ios/example/ios11/ios/RunnerTests/GoogleMapsPolylinesControllerTests.m

```diff
@@ -0,0 +1,57 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import google_maps_flutter_ios;
+@import google_maps_flutter_ios.Test;
+@import XCTest;
+@import GoogleMaps;
+
+#import <OCMock/OCMock.h>
+#import "PartiallyMockedMapView.h"
+
+@interface GoogleMapsPolylinesControllerTests : XCTestCase
+@property(strong, atomic) FLTGoogleMapPolylineController *polylineController;
```

Comment:
Fixtures should not be stateful without a very compelling reason; this can just be a local variable in each method, and `setUp` can become a utility method that creates and returns it (e.g., `controllerWithMockedMap`).

Reply:
Wasn't really sure how to share state between `setUp` and other functions without making them stateful. Just made it a kind of helper function instead of the overriden setup. The annoying part is that now other tests must call this function manually.

Reply:
> The annoying part is that now other tests must call this function manually.

That is a feature, as it makes interdependencies explicit and local. See https://abseil.io/tips/122

Reply:
That makes a lot of sense thinking more about. 🔥